addon/mixins/has-behaviors.js

```diff
@@ -0,0 +1,270 @@
+import Ember from 'ember';
+import withBackingField from 'ember-light-table/utils/with-backing-field';
+import ekbListenerName from 'ember-keyboard/utils/listener-name';
+
+const { A, observer, on } = Ember;
+
+/*
+ * from ember-keyboard
+ */
+const gatherKeys = function gatherKeys(event) {
```

Comment:
Ah that was my misunderstanding. I thought you took the entire file from ember-keyboard.